### PR TITLE
docstring changes and write exec for compile

### PR DESCRIPTION
**Pull request by JackTemaki**
Just some minor stuff I kept in my stashes for a long time.

### CODE REVIEW

File: returnn/training.py [hunk 1]
```diff
@@ -21,6 +22,8 @@
 class ReturnnModel:
     """
     Defines a RETURNN model as config, checkpoint meta file and epoch
+
+    This is deprecated, used Checkpoint instead.
```
Comment (michelwi) [inline returnn/training.py:26]:
```suggestion
    This is deprecated, use Checkpoint instead.
```
typo

File: returnn/training.py [hunk 2]
```diff
@@ -11,6 +11,7 @@
 import subprocess as sp
 
 from sisyphus import *
+from sisyphus.delayed_ops import DelayedSlice
```
Comment (michelwi) [inline returnn/training.py:14]: where is this needed? You only change doc strings so far.
```suggestion
```

Comment (JackTemaki) [reply returnn/training.py:14]: ah yes, did not see that one. Not sure what I used that for...

**Review by michelwi — APPROVED** [review]

File: returnn/training.py [hunk 3]
```diff
@@ -21,6 +21,8 @@
 class ReturnnModel:
     """
     Defines a RETURNN model as config, checkpoint meta file and epoch
+
+    This is deprecated, use Checkpoint instead.
```
Comment (Atticus1806) [inline returnn/training.py:25]:
```suggestion
    This is deprecated, use :class:`Checkpoint` instead.
```

**Review by Atticus1806 — COMMENTED** [review]
For some reason my PyCharm does not find the declarations, still I think we should reference the classes with the macro here, maybe in more parts of the file.

**Review by Atticus1806 — APPROVED** [review]

**Pull request merged**